Require NsatzTactic: nsatz support for Z and Q (#12861)
coq/coq: doc/changelog/10-standard-library/12861-nsatz-tactic-instances.rst

- **Changed:**

>   ``Require Import Coq.nsatz.NsatzTactic`` now allows using :tacn:`nsatz` out
>   of the box on `Z` and `Q`; previously users would either have to supply their
>   own instances or else ``Require Import Coq.nsatz.Nsatz``, which would
>   needlessly transitively requires files declaring axioms used in the reals

Reviewer: ```suggestion
  ``Require Import Coq.nsatz.NsatzTactic`` now allows using :tacn:`nsatz`
  with `Z` and `Q` without having to supply instances or using ``Require Import Coq.nsatz.Nsatz``, which
  transitively requires unneeded files declaring axioms used in the reals
```

If that doesn't change the meaning.
